### ISSUE TITLE
adjusting the configuration to reduce training time

### ISSUE THREAD

**Issue by RuyiLian** [THU-DA-6D-Pose-Group/GDR-Net#13]
Hi,

Thanks for sharing your great work. I have tried to train on LM and get results similar to your paper. The training process costs about 11 hours on my machine with single GPU. 

Now I would like to reduce the training time, e.g. increasing batch size to 64 and reducing the epochs to 80, but the performance drops significantly, especially for rotation (2° drops to 39.50). I think I may need to adjust other parameters too. Have your tried other configurations and could you give some suggestions?

**Comment by wangg12** [comment]
Sorry, I did not try reducing the number of epochs. Maybe you can tune the learning rate a little bit.

**Comment by RuyiLian** [comment]
Hi,

I found out that loss_region remains a large value (about 15 instead of 1 with original setting), though other loss terms are almost close to zero. 
![image](https://user-images.githubusercontent.com/54995948/123817185-ef4eae00-d8c5-11eb-82c3-39264c7445e7.png)

Based on the discussion in #11 I guess using large batch size will make this regularizer term harder to learn. I will try to alter other hyper parameters and see what will happen.
